packages/typespec-ts/src/index.ts

```diff
@@ -93,6 +93,14 @@ export async function $onEmit(context: EmitContext) {
       await calculateGenerationDir();
     dpgContext.generationPathDetail = generationPathDetail;
     const options: RLCOptions = transformRLCOptions(emitterOptions, dpgContext);
+    const hasTestFolder = await fsextra.pathExists(
+      join(dpgContext.generationPathDetail?.metadataDir ?? "", "test")
+    );
+    options.generateTest =
+      options.generateTest === true ||
```

Comment:
I didn't see tests/samples generated in todo_non_branding project. Are we able to generate them correctly for non-branding projects?

Reply:
seems non-branding test gen are filtered.

Reply:
I didn't enable the tests/sample generation for non branded case. as I think it's not clear yet.